style.css

```diff
@@ -83,3 +82,322 @@ h2 {
+  font-weight: bold;
+  background-color: #ff6b00;
+  border: solid 1px #fff;
+  color: #fff;
+  font-family: Inter, "Roboto", sans-serif;
+  font-style: normal;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+}
+
+.btn:hover {
+  opacity: 0.8;
+}
+
+.tech-stack-1 {
+  display: flex;
+  left: 12px;
+  top: 10px;
+  font-family: Inter, "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  margin-bottom: 10px;
+}
+
+.tech-stack-1 li {
+  cursor: pointer;
+  display: inline-block;
+  padding: 10px 20px;
+  font-weight: bold;
+  border: solid 1px #8993a4;
+  color: #091e42;
+  font-family: Inter, "Roboto", sans-serif;
+  font-style: normal;
+  font-size: 15px;
+  line-height: 20px;
+}
+
+.tech-stack-2 {
+  display: flex;
+}
+
+.tech-stack-2 li {
+  cursor: pointer;
+  display: inline-block;
+  padding: 5px 10px;
+  font-weight: bold;
+  border: solid 1px #8993a4;
+  background-color: rgb(255, 246, 246, 0.4);
+  font-family: Inter, "Roboto", sans-serif;
+  color: #fff;
+  font-style: normal;
+  font-size: 12px;
+  margin-bottom: 10px;
+  line-height: 20px;
+}
+
+.Feature {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  text-align: center;
+  align-items: center;
+  padding: 10px;
+}
+
+.Feature h3 {
+  width: 327px;
+  height: 36px;
+  font-family: Inter, "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  color: #091e42;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-top: 12px;
+  margin-bottom: 12px;
+}
+
+.Feat-text {
+  width: 327px;
+  height: 147px;
+  top: 48px;
+  font-family: Inter, "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+  margin-bottom: 15px;
+  text-align: justify;
+}
+
+.about-section .btn {
+  align-self: flex-start;
+  margin-left: 80px;
+}
+
+.my-works {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  align-items: center;
+  justify-content: center;
+}
+
+.my-works h2 {
+  width: 327px;
+  border-bottom: 1px solid #606163;
+  height: 83px;
+  font-family: Inter, "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #1c1a19;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-top: 24px;
+  margin-bottom: 24px;
+}
+
+.works-section-a {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-content: center;
+  text-align: center;
+  align-items: center;
+}
+
+.wrapper {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+  padding: 20px;
+}
+
+.projects {
+  background-image: url(src/Img\ Placeholder-proj.svg);
+  background-repeat: no-repeat;
+  text-align: center;
+  display: grid;
+  flex-direction: column;
+  align-items: center;
```

Comment:
- [ ] I suggest that you align the content of the project with `flexbox` as per project requirements.